api/tests/shared/unit/infrastructure/datasources/learning-content/challenge-datasource_test.js

```diff
@@ -308,6 +308,62 @@ describe('Unit | Infrastructure | Datasource | Learning Content | ChallengeDatas
     });
   });
 
+  describe('#findFlashCompatibleWithoutLocale', function () {
+    beforeEach(function () {
+      sinon.stub(lcms, 'getLatestRelease').resolves({
+        challenges: [
+          challenge_competence1,
+          challenge_competence1_notValidated,
+          challenge_competence1_noSkills,
+          challenge_competence1_obsolete,
+          challenge_competence2,
+          challenge_web1,
+          challenge_web1_notValidated,
+          challenge_web2_en,
+          challenge_web3,
+          challenge_web3_archived,
+        ],
+      });
+    });
+
+    context('when not requesting obsolete challenges', function () {
+      it('should resolve an array of matching Challenges from learning content', async function () {
+        // when
+        const result = await challengeDatasource.findFlashCompatibleWithoutLocale();
+
+        // then
+        expect(_.map(result, 'id').sort()).to.deep.equal(
+          [
+            challenge_competence1.id,
+            challenge_competence2.id,
+            challenge_web2_en.id,
+            challenge_web3.id,
+            challenge_web3_archived.id,
+          ].sort(),
+        );
```

Comment:
Pour eviter le sort:
```suggestion
        expect(_.map(result, 'id')).to.have.members(
          [
            challenge_competence1.id,
            challenge_competence2.id,
            challenge_web2_en.id,
            challenge_web3.id,
            challenge_web3_archived.id,
          ],
        );
```